hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -586,6 +587,15 @@ synchronized long expireServer(final ServerName serverName, boolean force) {
     }
     moveFromOnlineToDeadServers(serverName);
 
+    // If server is in draining mode, remove corresponding znode
+    String drainingZnode = ZNodePaths.joinZNode(master.getZooKeeper().getZNodePaths().drainingZNode,
+      serverName.getServerName());
+    try {
+      ZKUtil.deleteNodeFailSilent(master.getZooKeeper(), drainingZnode);
+    } catch (KeeperException e) {
+      LOG.warn("Error deleting the znode for draining server " +  serverName.getServerName(), e);
```

Comment:
Can u change the log to say "Error deleting the draining znode for stopping server "+ serverName